packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -747,6 +748,18 @@ mixin WidgetInspectorService {
   /// The current [WidgetInspectorService].
   static WidgetInspectorService get instance => _instance;
   static WidgetInspectorService _instance = _WidgetInspectorService();
+
+  final ValueNotifier<bool> _isSelectMode = ValueNotifier<bool>(true);
+
+  /// Whether the inspector is in select mode.
+  ///
+  /// In select mode, pointer interactions trigger widget selection instead of
+  /// normal interactions. Otherwise the previously selected widget is
+  /// highlighted but the application can be interacted with normally.
+  ValueListenable<bool> get isSelectMode => _isSelectMode;
+  set selectMode(bool value) {
```

Comment:
I am somewhat surprised that this doesn't trigger the `avoid_setters_without_getters` lint since this setter doesn't have a getter by the same name.